homeassistant/components/tradfri/light.py

```diff
@@ -155,8 +190,10 @@ async def async_turn_off(self, **kwargs):
         else:
             await self._api(self._device_control.set_state(False))
 
-    async def async_turn_on(self, **kwargs):
+    async def async_turn_on(self, **kwargs: Any) -> None:
         """Instruct the light to turn on."""
+        if not self._device_control:
+            return None
```

Comment:
It's not supposed to return anything.
```suggestion
            return
```

Reply:
ups, corrected.